README.md

```diff
@@ -33,6 +34,8 @@ the ConfigMap object
 the Secret object
 - **PERSISTENT_DIRECTORY** (*default: /var/demo\_files*): The fully-qualified path to the directory mounted
 with the PersistentVolume
+- **CLOUD**: The cloud that this app is running on (ex: AWS).
+- **NAMESPACE**: Namespace that this application is deployed into. This is used for the S3 bucket.
```

Comment:
Still needed?

Reply:
Nope.  I removed it in the latest version.